sunpy/map/sources/soho.py

```diff
@@ -130,7 +130,7 @@ def __init__(self, data, header, **kwargs):
             self.meta['date_obs'] = self.meta['date-obs']
         self._nickname = self.instrument + "-" + self.detector
         self.plot_settings['cmap'] = plt.get_cmap('soholasco{det!s}'.format(det=self.detector[1]))
-        self.plot_settings['norm'] = ImageNormalize(stretch=source_stretch(self.meta, PowerStretch(0.5)))
+        self.plot_settings['norm'] = ImageNormalize(stretch=source_stretch(self.meta, LinearStretch()))
```

Comment:
Is this also the best stretch for the actual calibrated lasco data?

Reply:
Level 0.5 and Level 1 data don't look great with any amount of stretch since you really need to subtract a background model in order to see anything like a CME.  Slightly better choices might be, say PowerStretch(0.5), but the advantage is not great.